packages/frontend/lib/index.ts

```diff
@@ -78,6 +78,14 @@ export default class Nango {
         }
     }
 
+    /**
+     * Creates a new unauthenticated connection using the specified provider configuration key and connection ID
+     * @param providerConfigKey - The key identifying the provider configuration on Nango
+     * @param connectionId -  The ID of the connection
+     * @param connectionConfig - Optional. Additional configuration for the connection
+     * @throws If there is an authentication error
```

Comment:
To be fully accurate, this function returns a rejected promise in case of error. Same comment for other functions below

Reply:
@TBonnin , just updated my pull request.

Reply:
I think we can skip the generic `throws a rejected promise in case of an error` since this can be deducted from the return statemetn

Reply:
@TBonnin , I agree 🙏🏻, just updated my pull request.